experimental/perf/scripts/run_load_tests.py

```diff
@@ -24,13 +24,29 @@ def print_help():
 
 def run_test_case(test_case, log_folder):
     if test_case == "bench_log":
-        test_case_helper.bench_log(log_folder, 4, 10000)
+        loop_count = 4
+        step_count = 10000
+        print(
```

Comment:
suggestion to use a logger instead of print, this way you can multiplex it both to the screen and to a file if you don't have access to the screen

Reply:
added